interop/test-runner/main.go

```diff
@@ -530,22 +546,122 @@ func (config *ScriptActorConfig) ChangeCipherSuite() error {
 func (config *ScriptActorConfig) RunStep(index int, step ScriptStep) error {
 	switch step.Action {
 	case ActionCreateGroup:
-		client := config.ActorClients[step.Actor]
-		groupID := []byte(uuid.New().String())
-		req := &pb.CreateGroupRequest{
-			GroupId:          groupID,
-			CipherSuite:      config.CipherSuite,
-			EncryptHandshake: config.EncryptHandshake,
-			Identity:         []byte(step.Actor),
-		}
-		resp, err := client.rpc.CreateGroup(ctx(), req)
+		var params CreateGroupStepParams
+		err := json.Unmarshal(step.Raw, &params)
 		if err != nil {
 			return err
 		}
-		config.Log(step.Actor, "CreateGroup", req, resp)
 
-		config.stateID[step.Actor] = resp.StateId
-		config.StoreMessage(index, "group_id", groupID)
+		// Create the group
+		groupID := []byte(uuid.New().String())
+		{
+			client := config.ActorClients[step.Actor]
+			req := &pb.CreateGroupRequest{
+				GroupId:          groupID,
+				CipherSuite:      config.CipherSuite,
+				EncryptHandshake: config.EncryptHandshake,
+				Identity:         []byte(step.Actor),
+			}
+			resp, err := client.rpc.CreateGroup(ctx(), req)
+			if err != nil {
+				return err
+			}
+			config.Log(step.Actor, "CreateGroup", req, resp)
+
+			config.stateID[step.Actor] = resp.StateId
+			config.StoreMessage(index, "group_id", groupID)
+		}
+
+		// If there are now members to add, we're done
+		if len(params.Members) == 0 {
+			return nil
+		}
+
+		// Get key packages from the joiners
+		keyPackages := make([][]byte, len(params.Members))
+		transactionIDs := make([]uint32, len(params.Members))
+		for i, member := range params.Members {
+			client := config.ActorClients[member]
+			req := &pb.CreateKeyPackageRequest{
+				CipherSuite: config.CipherSuite,
+				Identity:    []byte(member),
+			}
+			resp, err := client.rpc.CreateKeyPackage(ctx(), req)
+			if err != nil {
+				return err
+			}
+			config.Log(member, "CreateKeyPackage", req, resp)
```

Comment:
Should the CreateKeyPackage action be updated to also `Log` instead of `Store` private stuff?

Reply:
It already logs `Log`s the private stuff, in `CreateKeyPackageResponse`.  (Otherwise the `active-to-passive.js` script wouldn't work.)  But yeah, we can probably delete the `Store` there.  In general, we only need to `Store` things that will be used in later steps.

Reply:
Yeah, I meant delete the redundant `Store`. We can do this with the next PR.